packages/yoga/src/Stepper/native/Dots.jsx

```diff
@@ -17,67 +17,81 @@ const DotWrapper = styled.View`
 const Dot = styled.View(
   ({
     active,
+    secondary,
     theme: {
       yoga: {
         components: { stepper },
       },
     },
-  }) => `
-    width: 15px;
-    height: 15px;
+  }) => {
+    const state = secondary ? 'secondary' : 'active';
 
-    margin-top: -6px;
+    return css`
+      width: 15px;
+      height: 15px;
 
-    border-radius: ${stepper.dot.radius}px;
-    background-color: ${
-      active
-        ? stepper.dot.backgroundColor.active
-        : stepper.dot.backgroundColor.inactive
-    };
-  `,
+      margin-top: -6px;
+
+      border-radius: ${stepper.dot.radius}px;
+      background-color: ${active
+        ? stepper.dot.backgroundColor[state]
+        : stepper.dot.backgroundColor.inactive};
+    `;
+  },
 );
 
 const Label = styled(Text.Bold)(
   ({
     active,
+    secondary,
     theme: {
       yoga: {
         components: { stepper },
       },
     },
-  }) => `
-    width: 95px;
-    margin-top: 10px;
-    margin-left: -40px;
+  }) => {
+    const state = secondary ? 'secondary' : 'active';
 
-    color: ${
-      active ? stepper.label.color.active : stepper.label.color.inactive
-    };
+    return css`
+      width: 95px;
+      margin-top: 10px;
+      margin-left: -40px;
 
-    font-size: ${stepper.label.font.size}px;
-    text-align: center;
-  `,
-);
+      color: ${active
+        ? stepper.label.color[state]
+        : stepper.label.color.inactive};
 
-const Dots = ({ activeStep, labels }) => (
-  <Wrapper>
-    {labels.map((label, index) => (
-      <DotWrapper key={label}>
-        <Dot active={activeDot(index, activeStep)} />
-        <Label active={activeDot(index, activeStep)}>{label}</Label>
-      </DotWrapper>
-    ))}
-  </Wrapper>
+      font-size: ${stepper.label.font.size}px;
+      text-align: center;
+    `;
+  },
 );
 
+function Dots({ activeStep, labels, secondary }) {
+  return (
+    <Wrapper>
+      {labels.map((label, index) => (
+        <DotWrapper key={label}>
```

Comment:
I know it's rare, but assuming labels are always unique may be a problem, so it would review the component key here.
```suggestion
        <DotWrapper key={`${index}_$({label}`}>
```